packages/data-explorer/src/components/dataView/TableRow.vue

```diff
@@ -1,19 +1,45 @@
 <template>
   <tr>
     <td v-if="isShop"><shopping-button :isSelected="isSelected" :id="id"></shopping-button></td>
+    <td v-if="!isShop">
+      <div class="btn-group" role="group" aria-label="row actions">
+        <button
+          class="btn btn-sm btn-link"
+          role="button"
+          @click="$eventBus.$emit('delete-item', id)"
```

Comment:
Nice trick. :)

Reply:
tnks, only one of the possible options, suggest to discuss these during the next meeting about the design